blog.html

```diff
@@ -0,0 +1,65 @@
+﻿<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="styles/main-style.css">
+  <link rel="stylesheet" media="screen and (max-width: 500px)" href="styles/width-small.css">
+  <link rel="stylesheet" media="screen and (min-width: 501px) and (max-width: 850px)" href="styles/width-medium.css">
+  <link rel="stylesheet" media="screen and (min-width: 851px)" href="styles/width-large.css">
+  <title>Dog Blog</title>
+</head>
+<body>
+  <header class="header-container">
+    <div class="header-item-1">
+      <img src="images/cart-icon.svg" alt="cart icon">
+      <input class="header-item-2" type="button" value="Checkout">
+      <h3 class="cart-total"></h3>
+    </div>
+    <div class="header-item-3"><img src="images/adoptadog-logo.svg" alt="dog logo"></div>
+    <h3 class="header-item-4 home-h">Seattle's priemer dog adoption agency</h3>
+  </header>
+  <nav>
+    <ul class="nav-ul">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="dogs.html">Dogs</a></li>
+      <li><a href="blog.html">Dog Blog</a></li>
+      <li><a href="contact.html">Contact</a></li>
+    </ul>
+  </nav>
+  <section class="top-section">
+    <h1 class="home-h">Adoptadog Blog</h1>
+    <hr />
+    <section id="blog-content-id" class="top-section">
```

Comment:
You'll want to have this whole section empty, and generate the blog posts from JS. With jQuery it's not quite working, but regardless, this should be empty

Reply:
I emptied the section. the form is created by jQuery, and by completing an entry renders a post, also by jQuery, I also changed the jQuery to https link